internal/receiver/prometheusremotewritereceiver/internal/testdata/prometheus_remote_write_requests.go

```diff
@@ -0,0 +1,393 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testdata
+
+import (
+	"time"
+
+	"github.com/prometheus/prometheus/prompb"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+var (
+	Jan20 = time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC)
+)
+
+func SampleCounterTs() []prompb.TimeSeries {
```

Comment:
I don't recall seeing fixtures in testdata (static files). Can you move these to where they are used? Also since they are only used internally no need to export.